mappings/net/minecraft/tag/TagManager.mapping

```diff
@@ -1,4 +1,14 @@
 CLASS zd net/minecraft/tag/TagManager
+	CLASS zd$a PreparedBuilders
```

Comment:
A plural class name? This contains maps between ids and builders for each builder type, so it's really just a `IdentifierToBuilderMap`.

Reply:
This is really an intermediary object produced by prepare stage of reload. Hence `PreparedBuilders`. Or `TagReloadTask`? Need a better name indeed.

Reply:
It's not a task. It's just a list of maps between ids and builders:
```java
    public static class a {
        final Map<Identifier, Tag.Builder<Block>> a;
        final Map<Identifier, Tag.Builder<Item>> b;
        final Map<Identifier, Tag.Builder<Fluid>> c;
        final Map<Identifier, Tag.Builder<EntityType<?>>> d;

        public a(final Map<Identifier, Tag.Builder<Block>> map1, final Map<Identifier, Tag.Builder<Item>> map2, final Map<Identifier, Tag.Builder<Fluid>> map3, final Map<Identifier, Tag.Builder<EntityType<?>>> map4) {
            this.a = map1;
            this.b = map2;
            this.c = map3;
            this.d = map4;
        }
    }
```

The task just uses it.

Reply:
I say task for we have `Advancement.Task` "Task Advancement" which is similar to `Tag.Builder`.

Reply:
Should we name this `PreparedData` instead?

Reply:
`BuilderMap`/`BuilderMapContainer`/`BuilderContainer`?

Reply:
> Should we name this PreparedData instead?

Why PreparedData? It's a *public* class that stores all the builders for each tag.

Reply:
> [a *public* class](https://github.com/FabricMC/yarn/pull/674#discussion_r281057773)

Nah. Mojang makes a lot of unnecessary inner classes public, such as `CommandFunctionManager$Entry` and all those caching tags you just visited.

Reply:
Still, a container class should not be named based on its only usage.

Reply:
the names @Runemoro suggested make sense. I dont see many people using this class, but that doesnt need to limit its name.

Reply:
Using `BuilderHolder` now.